perspective.el

```diff
@@ -30,6 +30,7 @@
 ;;; Code:
 
 (require 'cl-lib)
+(require 'dash)
```

Comment:
I'm trying to keep Perspective dependency-free. Even dash is too much here, and its functionality should be reimplemented.

Reply:
I think we can use seq-intersection, or why not keep persp-current-buffers sorted by time in the first place.

Reply:
Please switch to `seq-intersection` and verify that it still works. If it breaks CI for older version of Emacs, we can maybe hack around the problem.